packages/plugins/typescript/resolvers/tests/ts-resolvers.spec.ts

```diff
@@ -622,9 +629,13 @@ __isTypeOf?: IsTypeOfResolverFn<ParentType, ContextType>;
 
     expect(mergedOutputs).not.toContain(`A: A;`);
     expect(mergedOutputs).not.toContain(`A: GQL_A;`);
+    expect(mergedOutputs).not.toContain(`C: GQL_MyC;`);
     expect(mergedOutputs).toContain(`NotMapped: GQL_NotMapped;`);
     expect(mergedOutputs).not.toContain(`NotMapped: NotMapped;`);
+    expect(mergedOutputs).toContain(`A: MyA;`);
     expect(mergedOutputs).toContain(`B: GQL_B;`);
+    expect(mergedOutputs).toContain(`C: C;`);
```

Comment:
Currently enums mentioned under `enumValues` doesn't comply `typesPrefix` (where other enums do comply. see `NotMapped` under this test).

Is this the ideal behavior? 🤔